iina/ThumbnailCache.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ThumbnailCache.swift
+//  iina
+//
+//  Created by lhc on 14/6/2017.
+//  Copyright © 2017 lhc. All rights reserved.
+//
+
+import Cocoa
+
+class ThumbnailCache {
+
+  static private let sizeofDouble = MemoryLayout<Double>.size
+  static private let sizeofInt64 = MemoryLayout<Int64>.size
+
+  static private let imageProperties: [String: Any] = [
+    NSImageCompressionFactor: 0.75
+  ]
+
+  static func exists(name: String) -> Bool {
```

Comment:
Actually it accepts a name (md5 value), not a path. I'll improve them according to your suggestions though.